.vscode/settings.json

```diff
@@ -1,12 +1,15 @@
 {
+    "python.analysis.extraPaths": ["./change_to_module_name", "./tests"],
+    "python.analysis.typeCheckingMode": "basic",
+    "python.languageServer": "Pylance",
     "python.linting.pylintEnabled": false,
     "python.linting.enabled": true,
     "python.linting.flake8Enabled": true,
     "python.linting.flake8Args": [
         "--max-line-length=110"
     ],
     "python.formatting.provider": "black",
-    "python.venvPath": "${workspaceFolder}/.venv/bin/python",
+    "python.venvPath": ".venv/bin/python",

```

Comment:
Now I see how this was useless. The new version is useless too, but at least it serves as a convenient template for new utilities.